qbft/spectest/tests/proposal/invalid_prepare_justification_round.go

```diff
@@ -18,9 +18,12 @@ func InvalidPrepareJustificationRound() *tests.MsgProcessingSpecTest {
 		testingutils.TestingPrepareMessageWithRound(ks.Shares[3], types.OperatorID(3), 2),
 	}
 	rcMsgs := []*qbft.SignedMessage{
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[1], types.OperatorID(1), 2),
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[2], types.OperatorID(2), 2),
-		testingutils.TestingRoundChangeMessageWithRound(ks.Shares[3], types.OperatorID(3), 2),
+		testingutils.TestingRoundChangeMessageWithRoundAndRC(ks.Shares[1], types.OperatorID(1), 2,
```

Comment:
why this change?

Reply:
This message has a justification on `main` branch (https://github.com/bloxapp/ssv-spec/blob/main/qbft/spectest/tests/proposal/invalid_prepare_justification_round.go#L43), but in a previous PR I made a mistake and removed it in some of the tests